include/tvm/relay/op_attr_types.h

```diff
@@ -210,7 +210,7 @@ using FForwardRewrite = runtime::TypedPackedFunc<Expr(
  *
  * \param orig_call the original Expr.
  * \param output_grad the gradient of the Expr.
- * \return the gradient for each parameters.
+ * \return the gradient for each parameter.
```

Comment:
Basically we’re not gonna touch Relay's codebase. So it is fine not changing them. But not a big issue.